app/global.css

```
@@ -167,3 +130,15 @@ code {
 }
+
+code {
+  overflow-x: scroll;
+}
+
+.url-snippet {
+  width: 75%;
+  margin-left: auto;
+  background: white;
+  padding: 5px;
+  margin: 5px 0;
```

Biome (1.9.4)

[error] Unexpected shorthand property margin after margin-left



(lint/suspicious/noShorthandPropertyOverrides)